targetingpacks.props

```diff
@@ -17,18 +17,12 @@
       <RuntimeReferencedAssemblyNames Include="Newtonsoft.Json" Condition="'$(SkipIncludeNewtonsoftJson)'!='true'" />
 
       <!-- Add Reference's to all files in the targeting pack folder, and to whitelisted items from the group above in the runtime folder. -->
-      <TargetingPackItems Include="%(TargetingPackDirs.Identity)/*" />
+      <TargetingPackItems Include="%(TargetingPackDirs.Identity)/*" Condition="'$(NoTargetingPackReferences)' != 'true'" />
       <Reference Include="%(TargetingPackItems.Filename)" Exclude="@(TargetingPackExclusions)"> <!-- TODO: System.Private.CoreLib shouldn't even be in the targeting pack. -->
         <Private>false</Private>
       </Reference>
-      <Reference Include="@(RuntimeReferencedAssemblyNames)" />
+      <ReferencePath Include="@(RuntimeReferencedAssemblyNames->'$(RuntimePath)%(Identity).dll')" />
```

Comment:
I think it can cause issues if you aren't careful what you reference. I think that this mechanism is going to be so rarely that it's not worth bothering with that.